src/aind_codeocean_utils/codeocean_job.py

```diff
@@ -0,0 +1,582 @@
+"""Module with generic Code Ocean job"""
+import logging
+import time
+from datetime import datetime
+from typing import Dict, List, Optional, Tuple, Union
+
+import requests
+from aind_codeocean_api.codeocean import CodeOceanClient
+from aind_codeocean_api.models.computations_requests import RunCapsuleRequest
+from aind_codeocean_api.models.data_assets_requests import (
+    CreateDataAssetRequest,
+    Source,
+    Sources,
+)
+
+LOG_FMT = "%(asctime)s %(message)s"
+LOG_DATE_FMT = "%Y-%m-%d %H:%M"
+
+logging.basicConfig(format=LOG_FMT, datefmt=LOG_DATE_FMT)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
We should probably let the user define the logger configurations outside of this library

Reply:
should I just leave `logging.getLogger()`?

Reply:
I think just leaving logging.getLogger should be fine